packages/core/src/components/Indicator/types.ts

```diff
@@ -0,0 +1,11 @@
+export interface IndicatorProps {
+  isVisible: boolean;
+  position?: 'right' | 'left';
+  background?: string;
+  text?: string | number | JSX.Element;
+  children?: React.ReactNode;
+}
+
+export interface ChildrenDataProps {
+  [key: string]: number;
+}
```

Comment:
Could we use `Record<string, number>` here?

Reply:
Done